fabric-datafixer-v1/src/main/java/net/fabricmc/fabric/impl/datafixer/mixin/MixinPersistentState.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.datafixer.mixin;
+
+import org.spongepowered.asm.mixin.Mixin;
+
+import net.minecraft.world.PersistentState;
+
+@Mixin(PersistentState.class)
+public class MixinPersistentState {
+	// Add our mod's data versions to method_17919 (INJECT @ after putInt)
```

Comment:
I fear this may blow up the size of these saved data files.

Reply:
Well based on the subclasses I see of PersistentState, the only one I would have a worry about having to implement this into would be ScoreboardState since objectives are a TypeReference the game can fix.